### PR TITLE
Feature/plot ci

### PR DESCRIPTION
**Pull request by zackt113**
Changed based on your comment. Ran a test, and all passed! Let me know how it looks now

### CODE REVIEW

File: src/strappy/display.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def plot_ci(sample, rep, bin_size=30, n="auto", ci_level=0.95,
     if path is not None:
         plt.savefig(path)
 
-    return (plt, title)
+    return (plt)
```
Comment (gauthampughazhendhi) [inline src/strappy/display.py]:
```suggestion
    return plt
```

**Comment by zackt113** [comment]
Thanks！

**Comment by zackt113** [comment]
pushed! @gauthampughazhendhi

File: src/strappy/display.py [hunk 2]
```diff
@@ -104,9 +107,9 @@ def plot_ci(sample, rep, bin_size=30, n="auto", ci_level=0.95,
     plt.ylabel(y_axis)
 
     if path is not None:
-        plt.savefig(path)
+        plt.savefig(path+"bootstrap_histogram.png")
```
Comment (gauthampughazhendhi) [inline src/strappy/display.py]:
```suggestion
        plt.savefig(f"{path}bootstrap_histogram.png")
```

**Comment by gauthampughazhendhi** [comment]
Looks good to me!

**Comment by zackt113** [comment]
> Looks good to me!

Thanks a lot, finally done! haha

**Pull request merged**